### PR TITLE
Add statuspage_io pack

### PR DESCRIPTION
**Pull request by jjm**
### Status
- new pack, work in progress
### Description

A pack for interacting with the statuspage.io service.
### Checklist (tick everything that applies)
- [X] Metadata: pack.yaml, icon, structure (required for new packs)
- [ ] Code linting (required, can be done after the PR checks)
- [ ] [Tests](https://docs.stackstorm.com/development/pack_testing.html) (not required but really recommended)

### CODE REVIEW

**Comment by tonybaloney** [comment]
Please can you squash your commits before this can be merged

**Comment by Kami** [comment]
@tonybaloney :laughing:

**Comment by tonybaloney** [comment]
@Kami ¯\_(ツ)_/¯

**Comment by tonybaloney** [comment]
@jjm new rules. didn't you get the memo?

**Comment by jjm** [comment]
@tonybaloney I don't think I can get it to 20+ commits ;-P so your most likely safe in 3rd place for commits.

**Comment by tonybaloney** [comment]
@jjm one of my packs has a bug, pretty sure I can fix that in <50 commits

File: packs/statuspage_io/CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# CONTRIBUTORS
+
+Jon Middleton <jon.middleton@pulsant.com>
```
Comment (Kami) [inline packs/statuspage_io/CONTRIBUTORS.md]: Just a heads up - in new version we added new `contributions` attribute to pack.yaml which is array so now more than one person can be list (most packs have more than one author / contributor these days) :)

Comment (jjm) [inline packs/statuspage_io/CONTRIBUTORS.md]: Cool. I'll make use of that soon 👍

**Comment by jjm** [comment]
I've not had time to work on finishing this pack and it's sometime before I will be able to get it into a useful & merge-able state.... 

So I think I'll just when we get to the point of StackStorm Exchange being ready.

**Comment by Kami** [comment]
Thanks fine.

If you don't mind, I will close this PR and you can open a new one against a new StackStorm-Exchange repo once it's up.

**Comment by jjm** [comment]
Closing, I'll come back to it later when StackStorm-Exchange is live.

**Pull request closed without merging**